cmd/webhook/main.go

```diff
@@ -213,6 +215,22 @@ func main() {
 		SecretName:  secretName,
 	})
 
+	mux := http.NewServeMux()
+
+	mux.HandleFunc("/", handler)
+	mux.HandleFunc("/health", healthHandler)
+	mux.HandleFunc("/readiness", readinessHandler)
+
+	// use PORT environment variable, or default to 8080
+	port := os.Getenv("PORT")
```

Comment:
What do you think about choosing a more descriptive env var name such as `PROBES_PORT` or similar?